spec/features/edit_form_can_take_info_and_update_station_spec.rb

```diff
@@ -0,0 +1,35 @@
+require_relative '../spec_helper'
+
+describe "When user clicks the edit button" do
+  it "They are taken to a new page" do
+    Station.create(name: "Station Name",
+                  city_id: City.find_or_create_by(name: "Denver").id,
```

Comment:
The idea behind `find_or_create_by` is to create something without making a duplicate if we already have it in our database. Since we're using DatabaseCleaner, don't believe we'll need it here since our database shouldn't have any cities in it at this point. Instead, you can use a regular old create.